okhttp/src/main/java/com/squareup/okhttp/TlsVersion.java

```diff
@@ -43,4 +43,12 @@ public static TlsVersion forJavaName(String javaName) {
     }
     throw new IllegalArgumentException("Unexpected TLS version: " + javaName);
   }
+
+  public static String[] javaNames(TlsVersion... tlsVersions) {
```

Comment:
nit: API wise, I think we need at least a method `javaName()` for a single instance call. Then this would be non-public, and possibly not in this file.